concourse/tasks/await-creds-in-secretsmanager.yml

```diff
@@ -11,7 +11,7 @@ inputs:
   - name: govuk-infrastructure
 params:
   AWS_REGION: eu-west-1
-  ASSUME_ROLE_ARN: 'arn:aws:iam::430354129336:role/govuk-concourse-deployer'
+  ASSUME_ROLE_ARN: ((concourse_deployer_role_arn))
```

Comment:
Does setting a var like `((var))` work in a task config file if you don't also provide the var to the task in the pipeline?

> The content of the config file may contain template ((vars)), which will be filled in using task step vars or a configured credential manager. [source](https://concourse-ci.org/jobs.html#schema.step.task-step.vars)

e.g.

```yaml
# in pipeline
- task: integration
  file: my-repo/ci/task.yml
  vars:
    text: "Hello World!"

# in task file
run:
  path: echo
  args: ["((text))"]
```

Maybe it works if this is set in [`SSM`](https://concourse-ci.org/aws-ssm-credential-manager.html)?

Reply:
I thought about it but keep it the same style as before because I find it nicer that all the dependent parameters are listed for a task in the `params` section. Either way will probably work.

Reply:
OK cool, I'm happy if it works! Please test it in your workspace before merging.